Box.V2/Models/Request/BoxEmailRequest.cs

```diff
@@ -7,7 +8,11 @@ namespace Box.V2.Models
     /// </summary>
     public class BoxEmailRequest
     {
+        /// <summary>
+        /// The level of access required for this folder upload email. Can be open or collaborators, or null which will be the default value
+        /// </summary>
         [JsonProperty(PropertyName = "access")]
-        public string Access { get; set; }
+        [JsonConverter(typeof(StringEnumConverter))]
+        public BoxEmailRequestAccessType? Access { get; set; }
```

Comment:
We should not change the type from string to Enum, since this is our public interface
@cburnette should we mark it with some annotation, so that we could correct them in the future?